modules/lvms-using-raid-arrays.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-using-raid-arrays_logical-volume-manager-storage"]
+= Integrating software RAID arrays with {lvms}
+
+You can create the Redundant Array of Independent Disks (RAID) array by using the `mdadm` utility, and integrate the RAID array with {lvms}. Logical Volume Manager (LVM) does not support creating a software RAID.
+
+You can integrate the RAID array with {lvms} while creating the `LVMCluster` custom resource (CR).
+
+.Prerequisites
+
+* You created a software RAID during the {product-title} installation.
++
+[IMPORTANT]
+====
+You can only create the RAID array during the installation of {product-title}.
+====
+
+* You have installed {lvms}.
+
+.Procedure
+
+. Open the `LVMCluster` CR YAML file.
++
+[NOTE]
+====
+If you have created the `LVMCluster` CR, edit the existing `LVMCluster` CR YAML file.
+====
```

Comment:
This `[NOTE]` is confusing because the step does not imply that the user is creating the file now. It says "Open the `LVMCluster` CR YAML file," which to me implies that it already exists.

Is there a file name/path or a command for opening this CR? If they might not have one yet, can you provide a command for creating a new one, or suggest a file name/location? I would also remove the note in favor of a step that combines both options, if both are valid (like "Create or edit...").

Reply:
This proc talks about adding a device path to the LVMCluster CR YAML file.
It can be done using the CLI or the web console at the time of creating the file and also after creating the file.

If the device path is added while creating the LVMCluster CR, we have two procs about creating LVMCluster CR using the CLI and web console.

If the device path is added after creating the LVMCluster CR, the user is trying to scale up the storage capacity. We have two procs about scaling up storage using the CLI and web console.

A couple of these modules are in progress. The plan was to add this module in the additional resources section of those relevant modules.

Reply:
Also, to create the LVMCluster CR, there are two steps:

1 .Create the LVMCluster CR as a YAML "file".
2. Create the LVMCluster CR by running the following command:
    oc create -f <file_name>

So in this case, users can perform the first step (create the YAML fle), and open it to modify as many times as needed before performing the second step.
Because "LVMCluster CR" won't be created until the second step is completed. 
In this case, it makes sense to use "Open the file". This step will also hold true in case if users have already created the "LVMCluster CR" (which means there is an existing LVMCluster CR YAML file).

@ousleyp Please let me know what you think.
Thanks.

Reply:
Thank you for the explanation! I understand what you mean by "Open..." making the most sense, because it doesn't assume where the user is in the process. I think what you have is good, and what was confusing me about it _is_ the fact that creating the file and creating the object are two different things (as you noted).

Here's a minor clarity suggestion that makes my brain slightly happier. Feel free to take it or leave it :) 

```
. Open the `LVMCluster` CR YAML file.
+
[NOTE]
====
If you already created the `LVMCluster` object, edit the existing `LVMCluster` CR YAML file.
====
```